ONSdigital/census-rh-ui
# Cr 1437 updates to enter room number

# Motivation and Context
Various tweaks and updates to 'Enter Room Number'

# What has changed
Adds recall of value if going back to 'Enter Room Number' after adding a value
Adds validation of character limit (design system code alerts user to over length value, but doesn't block it)
Updated and new unit tests to cover above

No Cucumber updates required

# How to test?
Via request journey, enter a CE address, then use 'Add a rom number' link to get to page
Try submitting a value over 10 characters and check for specific error message
Try submitting empty to check for empty message
Submit a valid value, then click the 'Change room number' link and confirm value previously submitted is in the input box

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1437

# Screenshots (if appropriate):
